scb-persistenceproviders/defectdojo-persistenceprovider/src/main/java/io/securecodebox/persistence/DefectDojoService.java

```diff
@@ -61,12 +62,16 @@
     protected String defectDojoDefaultUserName;
 
     protected static final String DATE_FORMAT = "yyyy-MM-dd";
+    protected static final String DATE_TIME_FORMAT = "yyyy-MM-dd hh:m:ss";
 
     Clock clock = Clock.systemDefaultZone();
 
     private String currentDate() {
         return LocalDate.now(clock).format(DateTimeFormatter.ofPattern(DATE_FORMAT));
     }
+    private String currentDateTime() {
```

Comment:
Does DefectDojo now support more detailed dates (with hour and minutes) for the tests?
If i remember correctly we tried to add them like this but it didn't used to work.

Reply:
No, only tests needs to be defined with time. As you mostly create it with an engagement, you didn't had contact so far.